### PR TITLE
link to Red Hat Container Catalog in README (#65)

### PR DESCRIPTION
**Pull request by vorburger**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -2,46 +2,68 @@
...
-### RHEL JDK 11
+#### RHEL JDK 11
 
 ```
 cekit --overrides-file=rhel7-jdk11-overrides.yaml build
 ```
 
 This will produce an image named `openjdk/openjdk-11-rhel7`
 
+### Testing
```
Comment (jmtd) [inline README.adoc]: Can we drop this section for now please

File: README.adoc [hunk 2]
```diff
@@ -2,46 +2,68 @@
 
 This is the OpenShift Source to Image (S2I) image for OpenJDK.
 
+## How to get it
+
+### From https://access.redhat.com/containers/[Red Hat Container Catalog]
+
+This is the subscription supported product version.  It's but free and a subscription is not required to get the image.
```
Comment (jmtd) [inline README.adoc]: I'm not sure what the "legal" or support story is for accessing the product images w/o subscription. I think the lack of authentication on that registry is more of a technical-limitation/legacy issue than a policy one (not sure though, although the newer registry registry.redhat.io does require authentication).  So I'd prefer to drop the sentence " It’s but free and a subscription is not required to get the image."

**Review by jmtd — CHANGES_REQUESTED** [review]

**Comment by jmtd** [comment]
Thanks for this. I'd like to merge it subject to the two review comments.

File: README.adoc (hunk 2 above)
Comment (vorburger) [reply README.adoc]: done

File: README.adoc (hunk 1 above)
Comment (vorburger) [reply README.adoc]: done

**Comment by vorburger** [comment]
> Thanks for this. I'd like to merge it subject to the two review comments.

Thanks; I've taken the two review comments into account.

**Pull request merged**